torch/storage.py

```diff
@@ -191,10 +191,16 @@ def complex_float(self):
         """Casts this storage to complex float type"""
         return self._to(torch.cfloat)
 
-    def pin_memory(self):
+    def pin_memory(self, pin_memory_device="cuda"):
```

Comment:
Can we just call this `device`? Thanks.

Reply:
OK，this modification has been done. Please take a moment to review it, thanks very much.